owncloud/fileInfo.js

```diff
@@ -13,7 +13,7 @@ function fileInfo(name, type, attr) {
 	this.fileInfo = {};
 
 	for (var key in attr) {
-		this.fileInfo[key.split(':')[1]] = attr[key];
+		this.fileInfo[key] = attr[key];
```

Comment:
I'm surprised that everything else still works ? If you keep the namespace then you need to adjust your `FileInfo` class to parse the values from the array using the namespace notation